packages/edit-site/src/components/dataviews/filter-summary.js

```diff
@@ -139,7 +139,16 @@ export default function FilterSummary( { filter, view, onChangeView } ) {
 				{ filter.operators.length > 1 && (
 					<DropdownSubMenu
 						trigger={
-							<DropdownSubMenuTrigger>
+							<DropdownSubMenuTrigger
+								suffix={
+									<>
+										{ filterInView.operator === OPERATOR_IN
+											? __( 'Is' )
+											: __( 'Is not' ) }
+										<Icon icon={ chevronRightSmall } />{ ' ' }
```

Comment:
Should we check for isRTL in all these chevronRight usage?

Reply:
I know that we have automatic inversions for CSS but so far, for icons we had to do it manually, so we should be consistent: either have the components handle this everywhere or not, because otherwise it will confuse developers

Reply:
Sorry, I wasn't clear – "everything should be reversed" within the context of menu items. The newer menu component doesn't need a chevron icon passed through as a suffix at all; that bit is automatic if the the menu item has a child menu.

Otherwise, yes, [RTL icons aren't always just flipped versions of their LTR equivalents](https://developer.apple.com/design/human-interface-guidelines/right-to-left#Interface-icons), and we should leave that decision to developers.

<p align="center">
  <img src="https://github.com/WordPress/gutenberg/assets/159848/63beee3b-fb15-449e-b0b9-fdc9e7c3472b" alt="A silhouette of a speaker pointing right with a backslash on top of it." width="100">
  <img src="https://github.com/WordPress/gutenberg/assets/159848/bdb8673d-7493-4b6d-8862-a9491d062ae3" alt="A silhouette of a speaker pointing left with a backslash on top of it." width="100">
</p>

Reply:
Mostly unrelated, but as a side-note, it would be great if the icon system was a bit smarter, and could handle bi-directionality implicitly. So rather than developers having to explicitly think about context...

```js
if ( isRTL() ) {
```

<img src="https://github.com/WordPress/gutenberg/assets/159848/bced35eb-9425-44bb-8e9c-8bb5fa2a5014" alt="A pilcrow next to a left-pointing arrow." align="left" height="48px">

```jsx
<Icon icon={ icons.formatRtl } ... />
```

```js
} else {
```

<img src="https://github.com/WordPress/gutenberg/assets/159848/3748483d-4e8b-4f15-88c0-bc7edc35a008" alt="A pilcrow next to a right-pointing arrow." align="left" height="48px">

```jsx
<Icon icon={ icons.formatLtr } ... />
```

```js
}
```

...they could just do something like this...

<img src="https://github.com/WordPress/gutenberg/assets/159848/bced35eb-9425-44bb-8e9c-8bb5fa2a5014" alt="A pilcrow next to a left-pointing arrow." align="left" height="48px">

```jsx
<Icon icon={ icons.format } ... /> {/* RTL */}
```

<img src="https://github.com/WordPress/gutenberg/assets/159848/3748483d-4e8b-4f15-88c0-bc7edc35a008" alt="A pilcrow next to a right-pointing arrow." align="left" height="48px">

```jsx
<Icon icon={ icons.format } ... /> {/* LTR */}
```

`</brain-dump>`

Reply:
Would something like this be too rudimentary?

```
[dir=ltr] {
    --icon-direction: 1;
}

[dir=rtl] {
    --icon-direction: -1;
}

.icon {
    transform: scaleX(var(--icon-direction));
}
```

Reply:
> Would something like this be too rudimentary?

Yes, because it doesn't always work.

<p align="center">
  <img src="https://github.com/WordPress/gutenberg/assets/159848/3748483d-4e8b-4f15-88c0-bc7edc35a008" alt="A pilcrow next to a right-pointing arrow." height="96px">
  <img src="https://github.com/WordPress/gutenberg/assets/159848/bced35eb-9425-44bb-8e9c-8bb5fa2a5014" alt="A pilcrow next to a left-pointing arrow." height="96px">
  <img src="https://github.com/WordPress/gutenberg/assets/159848/03b54142-0280-4f26-bc09-680416998c1d" alt="A" height="96px">
</p>

We could be a bit smarter in the SVG markup though, if we wanted to keep it out of the JS...

```svg
<SVG xmlns="http://www.w3.org/2000/svg" viewBox="-2 -2 24 24">
  <Path data-dir="ltr" d="..." />
  <Path data-dir="rtl" d="..." />
</SVG>
```

```css
[dir=rtl] .icon [data-dir=ltr],
[dir=ltr] .icon [data-dir=trl] {
  display: none;
}
```